packages/replay/src/integration.ts

```diff
@@ -136,6 +141,22 @@ export class Replay implements Integration {
       _experiments,
     };
 
+    if (typeof _experiments.mutationBreadcrumbLimit === 'number') {
```

Comment:
IMHO I would not do this, we don't usually do this for experiments, and this is not "critical" (and/or the semantics changed)?